code/modules/research/designs/wiremod_designs.dm

```diff
@@ -283,6 +283,11 @@
 	id = "comp_list_literal"
 	build_path = /obj/item/circuit_component/list_literal
 
+/datum/design/component/list_assoc_literal
```

Comment:
need to add this to the research node or else it won't be accessible

Reply:
Apparently /datum/design/component designs are base level?  I don't think the normal lathes can print these parts, only the wiremod lathe so its all put though there.  This shows up at round start as the other components.

Reply:
are you entirely sure about that? I'm checking the code right now and that really isn't the case. You need to go to the all_nodes.dm file and add it into the `/datum/techweb_node/basic_circuitry`

Reply:
It's very likely you are just adminned when you join and you're checking the component menu rather than the component printer because the component menu will show every single circuit component that exists (including inaccessible and admin only ones) if the person opening it is an admin.

Reply:
AHHH ok, sorry.  I didn't realize we use design_ids.  I could of sworn it did a search of /datum/design types.